.github/maven_prepare.yml

```diff
@@ -0,0 +1,32 @@
+#############################################################################
+# GitHub Action to Run Maven Release Prepare.
+#
+#############################################################################
+name: "Maven Release Prepare"
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'New Version'
+        required: true
+permissions:
+  contents: write
+  pull-requests: write
```

Comment:
Can you remove it from all yml that do nothing with PR creation

Reply:
Changes Done